### ISSUE TITLE
Allow number values in hide sms border

### ISSUE THREAD

**Issue by Augusto7743** [negativeExponent/smsplus-gx_libretro#42]
Hello.
Thanks for reading my topic.
SMS Plus GX allow "hide sms left border", but for some games is removed much area.
Not have option to select an value for tha overscan setting.
Please allow select an value for "hide sms left border" so as Nestopia does.

Have an nice week.

**Comment by gameblabla** [comment]
Which games are affected by this ? The "hide sms left border" is a setting that's intentionally set by games themselves.

**Comment by negativeExponent** [comment]
true. the border, from what i know is set by console itself and its either crop 8 on left or non at all.

**Comment by Augusto7743** [comment]
Thanks for replies.
Somes games when using "hide sms left border" crop the left screen extremely at point remove screen area for items in some games.
Alex Kidd in Miracle World is an example using that setting is cropped very much.
If using an manually others values avoid that issue.

**Comment by negativeExponent** [comment]
The option is not intended to be use the way normal overscan works with other consoles to simulate overscan on a tv screen. It is solely there to 'hide' the black borders that is set by the console itself on some of it's games.
